scripts/export-workflow-parameters.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+envprop() {
```

Comment:
```suggestion
# Read a property safely from config file, instead of sourcing the file
envprop() {
```

Kanskje også ta med kommentaren når vi flytter denne?